Library/Formula/glibmm.rb

```diff
@@ -18,6 +16,21 @@ class Glibmm < Formula
 
   def install
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    system "make"
+    system "make", "install"
```

Comment:
I am on holiday right now. Will make the necessary changes on Wednesday



Op 19-apr.-2015 om 23:21 heeft Jack Nagel <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


In Library/Formula/glibmm.rb<https://github.com/Homebrew/homebrew/pull/38741#discussion_r28657041>:

> @@ -18,6 +16,21 @@ class Glibmm < Formula
>
>    def install
>      system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
> -    system "make install"
> +    system "make"
> +    system "make", "install"


Otherwise, just change these two lines to just system "make", "install"

—
Reply to this email directly or view it on GitHub<https://github.com/Homebrew/homebrew/pull/38741/files#r28657041>.